src/diffpy/utils/diffraction_objects.py

```diff
@@ -215,73 +215,6 @@ def id(self):
     def id(self, _):
         raise AttributeError(_setter_wmsg("id"))
 
-    def set_angles_from_list(self, angles_list):
```

Comment:
For the following methods, I've double checked that no tests associated were found and pytest runs locally without these.